src/main/java/culturemedia/controllers/CultureMediaController.java

```diff
@@ -0,0 +1,26 @@
+package culturemedia.controllers;
+
+import java.util.*;
+
+import culturoteca.exception.VideoNotFoundException;
+import culturoteca.model.Video;
+import culturoteca.service.impl.CultureMediaServiceImpl;
+
+public class CultureMediaController {
+
+	private final CultureMediaServiceImpl cultureMediaService;
+
+
+	public CultureMediaController(CultureMediaServiceImpl cultureMediaService) {
+		this.cultureMediaService = cultureMediaService;
+	}
+
+
+	public List<Video> find_allVideos() throws VideoNotFoundException {
```

Comment:
The metod name find_allVideos()  shouldn't have low bar and It not used polymorphism.